src/bundle/ui-dev/src/modules/universal-discovery/components/filters/filters.row.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { createCssClassNames } from '../../../common/helpers/css.class.names';
+
+const FiltersRow = ({ children, title, extraClasses }) => {
+    const className = createCssClassNames({
+        'c-filters-row': true,
+        [extraClasses]: true,
+    });
+
+    return (
+        <div className={className}>
+            <div className="c-filters-row__title">{title}</div>
+            {children}
+        </div>
+    );
+};
+
+FiltersRow.propTypes = {
+    children: PropTypes.node.isRequired,
+    extraClasses: PropTypes.string.isRequired,
```

Comment:
I doubt extraClasses should be treated as required, I think it was always optional to pass this specific prop in our system?